build.gradle

```diff
@@ -112,6 +112,13 @@ subprojects { project ->
             java {
                 googleJavaFormat()
             }
+            groovyGradle {
+                target '**/*.gradle'
+                greclipse()
+                indentWithSpaces(4)
+                trimTrailingWhitespace()
+                endWithNewline()
+            }
```

Comment:
This and the other change in this file are the only real changes, right? Everything else is `.gradle` files being reformatted?

Reply:
Yes, that's correct, everything else is just reformatting